src/native/fs/fs_napi.cpp

```diff
@@ -124,10 +229,15 @@ set_stat_res(Napi::Object res, Napi::Env env, struct stat &stat_res)
     res["ctime"] = Napi::Date::New(env, uint64_t(round(ctimeMs)));
     res["birthtime"] = Napi::Date::New(env, uint64_t(round(birthtimeMs)));
 
+    auto xattr = Napi::Object::New(env);
+    res["xattr"] = xattr;
+    for (auto it = xattr_res.begin(); it != xattr_res.end(); ++it) {
+        xattr.Set(it->first, it->second);
```

Comment:
Do we want to use Set or we can set it as all the other places here using obj[key] = val ?

Reply:
It's not the same. The other cases are std::map, this one is Napi::Object.